libcontainer/intelrdt/intelrdt.go

```diff
@@ -161,6 +162,7 @@ func NewManager(config *configs.Config, id string, path string) *Manager {
 	}
 	if _, err := Root(); err != nil {
 		// Intel RDT is not available.
+		logrus.Errorf("Intel RDT is not available: %v", err)
```

Comment:
I'm sorry to say that, I think we should remove this line now. Please see my comment as follows.

Reply:
If we want to keep this, we should return error in `update.go`, for example:
```go
if intelRdtManager == nil {
    return fmt.Erroff("Intel RDT manager creation failed: likely due to missing resctrl filesystem")
}
```